code/modules/unit_tests/supermatter_airflow.dm

```diff
@@ -0,0 +1,15 @@
+/datum/unit_test/supermatter/start()
+
+/datum/unit_test/supermatter/airflow_hit_first/start()
+    var/turf/pos = locate(101, 100, 1) // Nice place with a good atmosphere
+    var/mob/living/carbon/human/human = new(pos)
+    var/obj/machinery/power/supermatter/shard/shard = new(pos)
+    shard.airflow_hit(human)
+    assert_eq(human.stat, DEAD)
+
+/datum/unit_test/supermatter/airflow_hit_second/start()
+    var/turf/pos = locate(101, 100, 1) // Nice place with a good atmosphere
+    var/mob/living/carbon/human/human = new(pos)
+    var/obj/machinery/power/supermatter/shard/shard = new(pos)
+    human.airflow_hit(shard)
+    assert_eq(human.stat, DEAD)
```

Comment:
move line 14 and 15 to line 9 and remove `airflow_hit_second`. There's no need to make two tests here, you can merge them together

Reply:
ah, okay.

Reply:
doing that has resulted in it runtiming due to null.stat

What should I do?

Reply:
oh, right, because it was dusted. You can reuse the human var and do another new()

Reply:
ah, okay

Reply:
finally, aha.